pkg/driver/agent.go

```diff
@@ -114,13 +114,28 @@ func registerCollectors(disableExporterMetrics bool) (*prometheus.Registry, erro
 			return nil, err
 		}
 	}
-	lvmCollector := collector.NewLvmCollector()
 
-	err := registry.Register(lvmCollector)
+	LvmVgCollector := collector.NewVgCollector()
```

Comment:
```suggestion
	lvmVgCollector := collector.NewVgCollector()
```

starting with smaller case is preferred.